src/shared/utils/reactProdInvariant.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule reactProdInvariant
+ */
+'use strict';
+
+/**
+ * WARNING: DO NOT manually require this module.
+ * This is a replacement for `invariant(...)` used by the error code system
+ * and will _only_ be required by the corresponding babel pass.
+ * It always throw.
+ */
+function reactProdInvariant(code, a, b, c, d, e, f) {
```

Comment:
Let's just drop the `a, b, c, d, e, f` here. (Esoteric fact: This actually affects `reactProdInvariant.length`.) Though you could probably also do `code, ...args` which might read a little nicer.